package.json

```diff
@@ -98,6 +99,7 @@
         "mqtt": "^5.10.1",
         "multer": "^1.4.5-lts.1",
         "nconf": "0.12.1",
+        "nyc": "^17.0.0",
```

Comment:
is this required for correlation ids?

Reply:
Nop don't know how this got there :/ will remove it